### PR TITLE
enhancement: Provide issue submission link for panics at cerbosctl

### LINKED ISSUE #282: Provide error submission links in case of an error from cerbosctl
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

We can provide new issue link in case of an error that has a bug potential. Eg. panic, internal errors. To demonstrate the proposal:

Let's say someone used `cerbosctl list` command, and then a panic happened in the code. We can provide some output like below:
```
$ cerbosctl list
It looks like an unexpected error happened. Would you mind reporting it to the link below?
https://github.com/cerbos/cerbos/issues/new/choose

goroutine 1 [running]:
runtime/debug.Stack()
        /opt/homebrew/Cellar/go/1.17/libexec/src/runtime/debug/stack.go:24 +0x88
main.main.func1(0x14000291900)
        /Users/serdar/git/cerbos/cmd/cerbosctl/main.go:87 +0xb0
        ...

exit status 1
```

### What would the ideal solution look like to you?

We can add a `recover()` check for panics and also search for internal server error message from the error strings in the `cmd.Execute()` call:

```Go
if err := cmd.Execute(); err != nil {
	if strings.Contains(err.Error(), "code = Internal") {
		...
	}
	...
}
```
### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by isdevx**
Signed-off-by: serdar <serdar@cerbos.dev>

#### Description

Fixes #282

#### Checklist 

- [x] The PR title has the correct prefix 
- [x] PR is linked to the corresponding issue
- [x] All commits are signed-off (`git commit -s ...`) to provide the [DCO](https://developercertificate.org/)

### CODE REVIEW

File: cmd/cerbosctl/internal/messages.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package internal
+
+const (
+	FeedbackMsg  = "It looks like an unexpected error happened. Would you mind reporting it to the link below?"
+	FeedbackLink = "https://github.com/cerbos/cerbos/issues/new/choose"
```
Comment (charithe) [inline cmd/cerbosctl/internal/messages.go:8]: How about generating a link that creates a specific type of issue with the appropriate title, label and body from the stacktrace? https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query

File: cmd/cerbosctl/internal/messages.go [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package internal
+
+const (
+	FeedbackMsg  = "It looks like an unexpected error happened. Would you mind reporting it to the link below?"
```
Comment (charithe) [inline cmd/cerbosctl/internal/messages.go:7]:
```suggestion
	FeedbackMsg  = "It looks like an unexpected error happened. Please use the link below to report it to Cerbos developers."
```

File: cmd/cerbosctl/internal/messages.go (hunk 1 above)
Comment (isdevx) [reply cmd/cerbosctl/internal/messages.go:8]: Sounds good but the content may be look a bit messy with the URL encoding. Nonetheless we can try it 👍

**Comment by isdevx** [comment]
Updating the link generation part, it will look like something similar to this:

resim

And the command output will look like this in case of a panic:
```
It looks like an unexpected error happened. Please use the link below to report it to Cerbos developers.

https://github.com/cerbos/cerbos/issues/new?body=Version%3A+unknown%0ACommit%3A+unknown%0AStack%3A%0A%60%60%60%0Agoroutine+1+%5Brunning%5D%3A%0Aruntime%2Fdebug.Stack%28%29%0A%09%2Fopt%2Fhomebrew%2FCellar%2Fgo%2F1.17.2%2Flibexec%2Fsrc%2Fruntime%2Fdebug%2Fstack.go%3A24+%2B0x88%0Amain.main.func1%280x14000388780%29%0A%09%2FUsers%2Fibrahim%2Fgit%2Fcerbos%2Fcmd%2Fcerbosctl%2Fmain.go%3A87+%2B0x74%0Apanic%28%7B0x1015bd160%2C+0x1016b4dd0%7D%29%0A%09%2Fopt%2Fhomebrew%2FCellar%2Fgo%2F1.17.2%2Flibexec%2Fsrc%2Fruntime%2Fpanic.go%3A1038+%2B0x21c%0Amain.main%28%29%0A%09%2FUsers%2Fibrahim%2Fgit%2Fcerbos%2Fcmd%2Fcerbosctl%2Fmain.go%3A95+%2B0x544%0A%0A%60%60%60%0A&labels=bug&title=bug%3A+Panic+in+cerbosctl

goroutine 1 [running]:
runtime/debug.Stack()
        /opt/homebrew/Cellar/go/1.17.2/libexec/src/runtime/debug/stack.go:24 +0x88
main.main.func1(0x14000388780)
        /Users/ibrahim/git/cerbos/cmd/cerbosctl/main.go:88 +0x150
panic({0x1015bd160, 0x1016b4dd0})
        /opt/homebrew/Cellar/go/1.17.2/libexec/src/runtime/panic.go:1038 +0x21c
main.main()
        /Users/ibrahim/git/cerbos/cmd/cerbosctl/main.go:95 +0x544
```

WDYT? cc/ @charithe

**Review by charithe — APPROVED** [review]
LGTM

**Pull request merged**